src/System.Security.Cryptography.Cng/src/Interop/NCrypt/NCrypt.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
```

Comment:
Why not src/Common/src/Interop/Windows/NCrypt/... ?  Since CNG is ultimately used in the Algorithms library (which has outstanding issues for having bad Interop style) there likely is P/Invoke signature sharing that could go on here.